docs/dev/new_check_howto.md

```diff
@@ -571,10 +571,17 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 ```
 
 **Windows** (Ensure that your shell session has _administrator_ privileges):
+
+For agent versions <= 6.11:
 ```
 "C:\Program Files\Datadog\Datadog Agent\embedded\agent.exe" integration install -w /path/to/wheel.whl
 ```
 
+For agent version >= 6.12:
+```
+"C:\Program Files\Datadog\Datadog Agent\bin\agent.exe" integration install -w /path/to/wheel.whl
```

Comment:
if we're going to have path `...bin\agent`, then path to wheel should also have `\`, yes? what does the integration command do with `/`?  Assume it's relative to `c:`?  or the current drive?

Note that also it's `c:\program....` by default, but doesn't have to be.  So we might want to indicate somehow
`<agent install dir>\bin\agent.exe...`

Reply:
I'm making the minimun change here